debezium-connector-postgres/src/test/java/io/debezium/connector/postgresql/RecordsStreamProducerIT.java

```diff
@@ -3033,6 +3065,26 @@ private SourceRecord assertRecordInserted(SourceRecord insertedRecord, String ex
         return insertedRecord;
     }
 
+    private SourceRecord assertRecordDeleted(String expectedTopicName, String pkColumn, Integer pk) throws InterruptedException {
+        assertFalse("records not generated", consumer.isEmpty());
+        SourceRecord insertedRecord = consumer.remove();
+
+        return assertRecordDeleted(insertedRecord, expectedTopicName, pkColumn, pk);
+    }
+
+    private SourceRecord assertRecordDeleted(SourceRecord deletedRecord, String expectedTopicName, String pkColumn, Integer pk) throws InterruptedException {
+        assertEquals(topicName(expectedTopicName), deletedRecord.topic());
+
+        // if (pk != null) {
```

Comment:
Why is this commented out? Is this related to the error you are mentioning?

Reply:
Yes, I uncommented this code